m4_sessions/1804inning/winning_through_failure.md

```diff
@@ -8,11 +8,43 @@ Maddy Dannenberg
 Steven Lee Lawson
 
 # Name of Session: 
+Winning Through Failure --choosing the path of most resistance 
 
 ## Session Description (Why should students come to this session):
 
+Learn how to accept failure as being okay and is part of growing as a person.
+
 ## Session Outcomes (What will students learn/leave with?):
 
+How to be comfortable talking about your failures in a positive light as it's a popular interview question.
+Here's how I faced these challenges, and how I came through it.
+
 ## Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+Intros: 15min
+  -Say your name and mod and a summer camp you went to and if you loved it or hated it, and why you chose this session.
+  
+
+  
+Stories of a few famous people of history (slides, video?) 4min
```

Comment:
Maybe start with your stories...or you can start in smaller groups and have each of you share.  Make it personal and then go historical.